prow/plugins/lifecycle/close.go

```diff
@@ -26,12 +26,16 @@ import (
 	"k8s.io/test-infra/prow/plugins"
 )
 
-var closeRe = regexp.MustCompile(`(?mi)^/close\s*$`)
+var (
+	closeRe           = regexp.MustCompile(`(?mi)^/close\s*$`)
+	closeNotPlannedRe = regexp.MustCompile(`(?mi)^/close\s+not-planned\s*$`)
```

Comment:
This regex will match the following which feels a little odd to allow.
```
/close




not-planned
```

If users are not allowed to have new lines between `/close` and `not-planned`, the regex needs to be something like 

```
regexp.MustCompile(`(?mi)^/close not-planned\s*$`) // allows only one space
(or)
regexp.MustCompile(`(?mi)^/close[ ]+not-planned\s*$`) // allowed one or more spaces (which feels a little weird too)
```

Reply:
Different plugins have different regexes, we should probably standardise the regex pattern. imo 
`regexp.MustCompile(`(?mi)^/close not-planned\s*$`)` sounds good - wdyt?, I'll also create an issue to track this.

Reply:
Sounds good to me.